server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/event/ComputeMessagePreviewListenerTest.java

```diff
@@ -171,6 +176,21 @@ void shouldKeepPreviewWhenCopyingMessage() throws Exception {
             .isEqualTo(PREVIEW);
     }
 
+    @Test
+    void shouldFailStoringPreviewWhenListenerHandlingException() throws Exception {
```

Comment:
I don't see value added by this test.

I can help you writing one test for testing ComputeMessagePreviewListener error handling if your need.